src/common/components/ui/ConfirmationModal.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+import { Modal, Button, ModalVariant, ButtonProps } from '@patternfly/react-core';
+
+type ConfirmationModalProps = {
+  title: string;
+  content: React.ReactElement | string;
```

Comment:
why not just 
```suggestion
  content: React.ReactNode;
```

Reply:
I can not tell, it's a secret.
Changed.